### ISSUE TITLE
Filter dead links

### ISSUE THREAD

**Issue by nandi95** [aquadzn/learn-x-by-doing-y#3]
It would be nice if there was a method devised that automatically removes dead links...

example link: https://github.com/aquadzn/learn-x-by-doing-y/blob/main/projects.csv#L125 returns the status code 410

Perhaps an action ran every (day/week) and remove lines from the csv file if a request to the endpoint fails?

**Comment by Tau5** [comment]
Maybe it would be better to include a link to the latest internet archive snapshot

**Comment by aquadzn** [comment]
> It would be nice if there was a method devised that automatically removes dead links...
> 
> example link: https://github.com/aquadzn/learn-x-by-doing-y/blob/main/projects.csv#L125 returns the status code 410
> 
> Perhaps an action ran every (day/week) and remove lines from the csv file if a request to the endpoint fails?

https://medium.com/we-are-orb/linear-regression-in-python-without-scikit-learn-50aef4b8d122 is working for me personally

I will try to add a script that checks the response code of the different links.

**Comment by nandi95** [comment]
> Maybe it would be better to include a link to the latest internet archive snapshot as most of the links are mosly text-only

Indeed, so it could be if the link's dead try the archive, if not just remove.
